src/main/java/client/UserClient.java

```diff
@@ -0,0 +1,161 @@
+package client;
+
+import dto.requests.ResponseInfo;
+import dto.requests.user.User;
+import io.qameta.allure.Step;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.apache.http.HttpStatus;
+
+import java.io.IOException;
+import java.util.List;
+
+public class UserClient extends BaseClient {
+    final private static String USER_ENDPOINT = "/v2/user/";
+    final private static String USER_LIST_ENDPOINT = USER_ENDPOINT + "createWithList";
```

Comment:
Remove the last "/" from USER_ENDPOINT and put it at the beginning of every new endpoint: `/createWithList`, `/createWithArray` etc